java/org/contikios/cooja/plugins/Visualizer.java

```diff
@@ -1278,7 +1278,7 @@ public void paintMotes(Graphics g) {
       int x = pixelCoord.x;
       int y = pixelCoord.y;
 
-      if (mote == movedMotes) {
+      if (movedMotes.contains(mote)) {
```

Comment:
`movedMotes` is currently unused and always `null`. I tried to trace this in the git history, and I think it is a remainder from when the option of moving single motes was replaced with mote multi-selection and the option to move multiple motes simultaneously. I think `movedMotes` should simply be removed as it seems to be unused.

Reply:
Even better, updated patch!

Reply:
Thanks for the update!

This change also makes the constant `MOVE_COLOR` unused so perhaps it should be removed it at the same time? Also, you might want to update the PR message?

Reply:
Fixed.

Reply:
Thanks!